src/components/Timer/timer.ts

```diff
@@ -0,0 +1,58 @@
+import { TrackPath, Challenge } from '../../model/ChallengeConfiguration'
+import * as turf from '@turf/turf'
+interface TrackPoint {
+  lat: Number
+  lon: Number
+  time: Date
+}
+// type TrackPath = TrackPoint[]
+interface StartStopLine {
+  firstPoint: TrackPath
+  endPoint: TrackPath
+}
+// interface Challenge {
+//   start: StartStopLine
+//   finish: StartStopLine
+// }
+interface Props {
+  path: TrackPath
+  challenge: Challenge
+}
+{
+  /* <LineIntersect path={pathfromUploadedFile} challenge={defaultChallenge} /> */
```

Comment:
🚧 I saw some ESLint warnings on a few places in the file, like here. Also, since this isn't using JSX Elements / React Components, we shouldn't see any HTML-like code like this. Let's fix the warnings that ESLint puts out by deleting this line, and looking at the other warnings

*Warnings are usually optional-ish, but Netlify has strict settings enabled that will cause it to error out on ESLint warnings like this, at least once this function is included in the build.*